errors.go

```diff
@@ -59,7 +59,7 @@
 // New, Errorf, Wrap, and Wrapf record a stack trace at the point they are
 // invoked. This information can be retrieved with the following interface.
 //
-//     type StackTrace interface {
+//     interface {
```

Comment:
What about the example cast that mentions the `StackTrace` type [a few lines down](https://github.com/pkg/errors/blob/45fa7bc37676ec7003f63dad4dda3955012806d8/errors.go#L74)? Should that change as well?